liquibase-couchbase/src/test/java/integration/statement/RemoveDocumentsQueryStatementIT.java

```diff
@@ -0,0 +1,80 @@
+package integration.statement;
+
+import com.couchbase.client.java.Collection;
+import com.couchbase.client.java.json.JsonObject;
+import com.couchbase.client.java.manager.query.CreatePrimaryQueryIndexOptions;
+import com.google.common.collect.Lists;
+import common.TransactionStatementTest;
+import common.operators.TestCollectionOperator;
+import liquibase.ext.couchbase.statement.RemoveDocumentsQueryStatement;
+import liquibase.ext.couchbase.types.Document;
+import liquibase.ext.couchbase.types.Id;
+import liquibase.ext.couchbase.types.Keyspace;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static common.constants.TestConstants.INDEX;
+import static common.matchers.CouchbaseCollectionAssert.assertThat;
+import static liquibase.ext.couchbase.types.Keyspace.keyspace;
+
+class RemoveDocumentsQueryStatementIT extends TransactionStatementTest {
+
+    private static final String DOC_FIELD_NAME = "field";
+    private static final String DOC_FIELD_Value = "val";
+    private final TestCollectionOperator collectionOperator = bucketOperator.getCollectionOperator(collectionName, scopeName);
+    private List<Id> ids;
+    private Document doc1;
+    private Document doc2;
+    private Document doc3;
+    private Collection collection;
+    private Keyspace keyspace;
+
+    @BeforeEach
+    @SneakyThrows
+    void setUp() {
+        keyspace = keyspace(bucketName, scopeName, collectionName);
+        collection = bucketOperator.getCollection(collectionName, scopeName);
+        collectionOperator.createPrimaryIndex(CreatePrimaryQueryIndexOptions
+                .createPrimaryQueryIndexOptions()
+                .indexName(INDEX));
```

Comment:
create primary index 1 time for all tests in this class. No point to create it and drop for each test

Reply:
Not fixed yet.

Reply:
Sorry, missed this part. Fixed now)